src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue27367.cs

```diff
@@ -0,0 +1,37 @@
+#if TEST_FAILS_ON_CATALYST && TEST_FAILS_ON_WINDOWS
+// On Catalyst, Swipe actions not supported in Appium.
+// [Windows] Swiping does not work with SwipeItemView in SwipeView , see https://github.com/dotnet/maui/issues/27436
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue27367 : _IssuesUITest
```

Comment:
Both tests are failing on Android with small snapshot differences. For example:
![image](https://github.com/user-attachments/assets/491e35c0-4010-4672-aeaa-ae931151e108)
Look wrong now, could you review it?

Reply:
Hi @jsuarezruiz ,

The test fails because the CI takes a screenshot before the swipe is fully completed, resulting in a slight visual difference. Adding a `WaitForElement` before VerifyScreenshot resolves the issue.